config/index.js

```diff
@@ -251,6 +251,22 @@ var conf = convict({
       format: String,
       default: undefined,
       env: 'SES_CONFIGURATION_SET'
+    },
+    bounces: {
+      hard: {
+        max: {
+          doc: 'Maximum number of hard bounces before blocking emails',
+          default: 0,
```

Comment:
We should have a way to quickly pref it off; perhaps having values <= 0 mean "no limit" is a simple way to achieve this.